devchat.go

```diff
@@ -195,6 +196,8 @@ func newUser(s ssh.Session) *user {
 		win:           w,
 		lastTimestamp: time.Now(),
 		joinTime:      time.Now(),
+		timezone:      nil,
```

Comment:
nil is the zero value so this line can be deleted

Reply:
Isn't it better to clearly state that having the timezone set to nil is an expected behavior and the default value?
Otherwise, someone might thought that letting that out is a bug.

Reply:
Hmm, I already leave out all zero values